protobuf/model_config.proto

```diff
@@ -1661,6 +1662,16 @@ message ModelConfig
   //@@
   int32 max_batch_size = 4;
 
+  //@@  .. cpp::var:: bool response_cache_enable
+  //@@
+  //@@     Whether or not to use response cache for the model. If True, the
+  //@@     responses from the model are cached and when identical request
+  //@@     is encountered, instead of going through the model execution,
+  //@@     the response from the cache is utilized. By default, response
+  //@@     cache is disabled for the models.
+  //@@
+  bool response_cache_enable = 24;
```

Comment:
To future proof, can you add a ModelResponseCache message that has a single "bool enable" entry for now. Then after model_repository_agents add "ModelResponseCache response_cache"